### DISCUSSION TITLE
How to find the correct repository for a tool or technology

### DISCUSSION THREAD

**Discussion by mlsomers** [community/community#51460] [Repositories]
### Select Topic Area

Question

### Body

I want to open an issue for this command-line tool from the .Net core SDK:

`dotnet test MyTests.dll -l:trx;LogFileName=Output.xml`

I asked it in the [wrong place](https://github.com/xunit/visualstudio.xunit/issues/368) and was forwarded to "the Azure Pipelines team", however when searching for the correct repository I quickly ran out of my search-quota and have to wait before being allowed to do new searches.

There should be a way to index repositories and find them on:
 - Commandline (or executable file name)
 - URL
 - Classname
 - Nuget package name
 
 Especially with .Net core related stuff I find it extremely difficult to find the correct repositories.

**Comment by Anmol-Baranwal** [comment]
![image](https://user-images.githubusercontent.com/74038190/229502700-3ddef300-45df-4797-a098-c9f7cbf7840f.png)

To find the correct repository for a command-line tool, URL, classname, or NuGet package name, you can try using GitHub's advanced search feature. Here's how:

1. Go to the GitHub homepage (github.com).
2. Click on the search bar at the top of the page.
3. Click on the "Advanced search" button to the right of the search bar.
4. In the advanced search page, you can enter keywords related to your search. For example, if you want to find the repository for the dotnet test command-line tool, you can enter "dotnet test" in the search bar.
5. You can also filter the search results by repository name, language, license, and other criteria. For example, you can filter the results to show only repositories that are written in C# or that have a certain number of stars.
6. Once you have entered your search criteria and filters, click on the "Search" button to see the results.

**Comment by mlsomers** [comment]
Thanks @Anmol-Baranwal , although I wasn't able to find the correct repository (i'm wondering if the tool is open source after all) but it seems like Google does a better job than the built in search.

It would still be cool if repositories could register executable filenames or DLL names in a field where they can be specifically searched for.
